src/main/frontend/handler/common.cljs

```diff
@@ -241,3 +233,14 @@
       (d/set-style! context-menu
                     :left (str client-x "px")
                     :top (str (+ scroll-y client-y) "px")))))
+
+(defn parse-config
+  "Parse configuration from file `content` such as from config.edn."
+  [content]
+  (try
+    (rewrite/parse-string content)
+    (catch js/Error e
+      (println "Parsing config file failed: ")
```

Comment:
Could we use `log/error` instead of print? With logging we can distinguish between debugging meant for production and debugging that wasn't removed during development

Reply:
Updated!